### ISSUE TITLE
Tool does not account for ObjectValues

### ISSUE THREAD

**Issue by RatRat12** [rojo-rbx/remodel#96]
I have ObjectValues under some models that reference other models, since the tool exports them as separate .rbxm files, they can't reference each other anymore, when I build the place back the values are empty. One workaround for now Is to export lowest common ancestor of the given models as a one .rbxm file instead of going down the hierarchy and exporting children separately

**Comment by eAi** [comment]
I’m not sure whether you’re reporting a bug here. I believe this behaviour is what you would expect. Doing anything else isn’t possible within the constraints of the format as it doesn’t support references outside a file - and the way remodel works wouldn’t really be compatible with this as each model file is generated one-by-one.

**Comment by RatRat12** [comment]
Yeah that does make sense, registering this as a bug would be pointless. But the documentation should highlight such caveats more clearly at least, or throw Rojo warning in the game output window for object values that failed to map

**Comment by eAi** [comment]
Yes, I agree about that. I think this is probably severe enough that remodel should error if this happens, at least by default, as tracking it down isn’t easy and it isn’t necessarily intuitive. I believe the references that are left in the file are invalid after this happens - they point to an ID that doesn’t exist.
